catalog.html

```diff
@@ -0,0 +1,256 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона</title>
+    <link rel="stylesheet" href="styles/styles.css">
+  </head>
+  <body>
+    <header class="page-header container">
+      <nav class="navigation">
+        <a class="logo" href="index.html">
+          <img src="images/logo-sedona.svg" width="138" height="64" alt="Логотип города Седона.">
+        </a>
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a class="navigation-link" href="index.html">Главная</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">О Седоне</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="catalog.html">Гостиницы</a>
+          </li>
+        </ul>
+        <ul class="navigation-user">
+          <li class="navigation-user-item">
+            <a class="navigation-search" href="#">
+              <span class="visually-hidden">Поиск</span>
+            </a>
+          </li>
+          <li class="navigation-user-item">
+            <a class="navigation-likes" href="#">
+              <span class="visually-hidden">Количество выбранных отелей</span>
+              <span class="number-of-selected-hotels">12</span>
+            </a>
+          </li>
+          <li class="navigation-user-item">
+            <a class="navigation-choice" href="#">
+              <span>Хочу сюда!</span>
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+      <main class="main-catalog container">
+        <div class="catalog-background-header">
+          <h1 class="catalog-title">Гостиницы Седоны</h1>
+          <ul class="breadcrumbs">
+            <li class="breadcrumbs-item">
+              <a class="breadcrumbs-link breadcrumbs-link-main" href="index.html">
+                <span class="visually-hidden">Вернуться на главную страницу</span>
+              </a>
+            </li>
+            <li class="breadcrumbs-item">
+              <p class="breadcrumbs-hotels">Гостиницы</p>
+            </li>
```

Comment:
```suggestion
            <li class="breadcrumbs-item">
                   Гостиницы
            </li>
```
`p` это лишняя обертка